src/ddmd/backend/cgobj.c

```diff
@@ -1724,11 +1724,14 @@ void Obj::staticdtor(Symbol *s)
 
 
 /***************************************
- * Stuff pointer to function in its own segment.
- * Used for static ctor and dtor lists.
+ * Set up function to be called as static constructor on program
```

Comment:
> Stuff pointer to function in its own segment. 

This is useful information.
please keep it.